platform/security/servlet/security-servlet-logout/src/main/webapp/js/main.js

```diff
@@ -40,7 +40,8 @@
                     .click(function () {
                         $('iframe').attr('src', action.url);
                         $('#modal').removeClass('is-hidden');
-
+                        window.localStorage.removeItem('systemUsage');
```

Comment:
Just to clarify, what is 'systemUsage' for, and why did we need to remove it here.